mock-builder/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## Release 0.2.0
+- Remove hashes in storages for pallet mocks
+
+## Release 0.1.2
+- Support for `polkadot-v1.0.0`
+
+## Release 0.1.1
+- Support for `polkadot-v0.9.43`
+
+## Release 0.1.0
+- Base functionality
+- Support for `polkadot-v0.9.38`
```

Comment:
This is my initial idea for tracking changes, but not sure at all.

- Do we expect to be referenced by the version or by a git+branch address?
- If choosing git+branch address, I think we're locked only to do breaking changes when the polkadot branch increases. Do we want that?

Reply:
So if last option chosen: CHANGELOG will be: 

```markdown
# Changelog

## Release 0.2.0
- Remove hashes in storages for pallet mocks

## Release 0.1.0
- Base functionality
```

And if this PR is merged over `polkadot-v1.0.0` and we want to offer this change, we could create a `polkadot-v1.0.0-2` branch user who wants to use it before polkadot 1.1.0

Reply:
I was wondering whether we can combine the `mock-builder` versioning more explicitly with the underlying Polkadot version, like using the Polkadot version as suffix of `v1.*` leading to `v1.0.9.43`, and `v1.1.{0, 1, 2}`.

Reply:
> I was wondering whether we can combine the mock-builder versioning more explicitly with the underlying Polkadot version, like using the Polkadot version as suffix of v1.* leading to mock-builder versions v1.0.9.43, and v1.1.{0, 1, 2}.

I think this is more or less what I said about having a branch with `polkadot-v0.9.43-x` in case we want to support a breaking feature of `mock-builder`, right?

> I propose to keep it simple for now and rely on branch referencing.

Agree with this 👍🏻